lib/form/class-editor.php

```diff
@@ -142,6 +142,7 @@ public function field_markup() {
 		 * @since 4.1.8
 		 */
 		$value = apply_filters( 'ap_editor_pre_value', $this->value() );
+		// error_log( 'value: '.$value.' id: '.$this->id() );
```

Comment:
Please remove this line.

Reply:
It is commented out, so will do no harm - do you have an internal logging method to write to the log file we can use for future testing?

Reply:
Still this has to be removed. :) Thanks you